docs/system/process-metrics.md

```diff
@@ -20,32 +20,191 @@ metrics](/docs/runtime/README.md#metrics).
 <!-- toc -->
 
 - [Metric Instruments](#metric-instruments)
-  * [Process](#process)
-- [Attributes](#attributes)
+  * [Metric: `process.cpu.time`](#metric-processcputime)
```

Comment:
Maybe consider making the section follow the same structure as in the system metrics markdown: https://github.com/braydonk/semantic-conventions/blob/process_metrics/docs/system/system-metrics.md?plain=1#L24